front/src/routes/QuestionsPage.svelte

```diff
@@ -90,14 +103,61 @@
         loading = false;
         showError();
       });
-  }
+  };
+
+  const openConfirmationModal = () => {
+    modalText = `Are you sure you want delete these questions?`;
+    modalHeadingText = 'Delete selected questions';
+    changeStatus = () => {
+      deleteQuestions();
+    };
+    confirmationModal = true;
+  };
+
+  const deleteQuestions = async () => {
+    loading = true;
+    confirmationModal = false;
+    selectedRowIds.forEach(async id => {
+      wretch(`${API_URL}/questions/${id}`)
+        .auth(`Bearer ${$token}`)
+        .options({ credentials: 'include', mode: 'cors' })
+        .delete()
+        .res(() => {
+          loading = false;
+        })
+        .catch(() => {
+          loading = false;
+          showError();
+        });
+
+      rows.splice(
+        rows.findIndex(row => {
+          return row.id === id;
+        }),
+        1
+      );
+    });
+
+    selectedRowIds = [];
```

Comment:
`loading = false` should be set here, now you set it to `false` right after first request